tests/test_config_and_args.py

```diff
@@ -174,13 +174,24 @@ def test_anyio_from_config(tmp_path: Path, capsys: pytest.CaptureFixture[str]):
         "subprocess.Popen",
         "[anyio|trio]",
     )
-    err_file = str(Path(__file__).parent / "eval_files" / "anyio_trio.py")
-    expected = f"{err_file}:10:5: ASYNC220 {err_msg}\n"
+    err_file = Path(__file__).parent / "eval_files" / "anyio_trio.py"
+
+    # find the line with the expected error
+    for i, line in enumerate(err_file.read_text().split("\n")):
+        if "# ASYNC220: " in line:
+            # line numbers start at 1, enumerate starts at 0
+            lineno = i + 1
```

Comment:
```suggestion
    for lineno, line in enumerate(err_file.read_text().split("\n"), start=1):
        if "# ASYNC220: " in line:
```

https://docs.python.org/3/library/functions.html#enumerate

Reply:
:heart_eyes: